spec/features/access_start_page_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature "Access Start Page" do
```

Comment:
can you pls make sure there is a ticket in jira to sort out the start page ?

Reply:
Yes, this test fails and there is a ticket - `RST-867`.